commitlint.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
js file for commitlint... maybe put it into package.json
at least use a consistent method to configure these tools